src/modules/teachers/actions/assignment.js

```diff
@@ -7,34 +7,55 @@ import * as types from '../../../constants/actionTypes';
 
 
 // Action creators
-const { root, assignments } = config.eduAPI;
+const { root, assignments, sampleSubjects, workflowId } = config.eduAPI;
 
 export function createAssignment(assignment, classroomId) {
+  let subjectData = [];
+  let metadata;
   const classroomData = {
     id: classroomId,
     type: 'classrooms',
   };
-  const metadata = {
-    classifications_target: assignment.classifications_target,
-    description: assignment.description,
-    duedate: assignment.duedate,
-    filters: assignment.filters,
-    subjects: assignment.subjects,
-  };
   const studentData = assignment.students.map(student_id => ({
     id: student_id,
     type: 'student_user',
   }));
-  const subjectData = assignment.subjects.map(subject_id => ({
-    id: subject_id,
-    type: 'subjects',
-  }));
-  
+
+  // Carto DB doesn't have staging data. We use hard coded subject ids from the staging WG project and ignore the Carto selection.
+  if (process.env.NODE_ENV === 'staging') {
```

Comment:
Should this check using the `env` variable from `config.js` instead of using process.env.NODE_ENV?

Reply:
Also, should it check for both `env === 'staging' || env === 'development'` ?